### PR TITLE
Added Back button in 'Course Type' Form in Course Creator

### PR DESCRIPTION
**Pull request by PR4NJ41**
## What this PR does
The Course Creator doesn't have a back button when we choose 'course type', making it difficult for users to propagate back and forth. This PR adds a back button for proper navigation while creating an independent program for better User Experience.

## Screenshots
Before:

https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/96368921/b496a84f-36bd-41b5-847a-3e1e627bd7b1

After:

https://github.com/WikiEducationFoundation/WikiEduDashboard/assets/96368921/f1e4efbe-efa9-4eca-a83b-8207d8afb435

### CODE REVIEW

**Comment by PR4NJ41** [comment]
@ragesoss This PR is ready for Review. Please review it. Thank You!

**Comment by PR4NJ41** [comment]
@ragesoss This PR is ready for Review. Please review it. Thank You!

File: app/assets/javascripts/components/course_creator/course_type.jsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ const CourseType = ({ wizardClass, wizardAction }) => {
           );
         })}
       </div>
+      <button className="button dark" onClick={back}>Back</button>
```
Comment (ragesoss) [inline app/assets/javascripts/components/course_creator/course_type.jsx:34]: This should use i18n.

Comment (PR4NJ41) [reply app/assets/javascripts/components/course_creator/course_type.jsx]: Done

**Comment by PR4NJ41** [comment]
@ragesoss This PR is ready for Review. Please review it. Thank You!

**Pull request merged**